front/src/App.js

```diff
@@ -28,6 +29,11 @@ class App extends Component {
                 )}
               </Col>
               <Col>
+                {localStorage.getItem("muzi_muzi_token") !== null ? (
+                  <Route path="/new-advert" component={AddMusicianAdvert} />
+                ) : (
+                  true
+                )}
                 <Route exact path="/" component={News} />
```

Comment:
Maybe put all the secured routes to a function that returns them al when user is logged in?